aries-site/src/layouts/content/Example.js

```diff
@@ -115,12 +115,14 @@ export const Example = ({
       <Box margin={{ vertical: 'small' }} gap="large">
         <Box>
           <Box
-            direction="row"
+            align={!template ? 'center' : undefined}
             background="background-front"
-            pad={template ? { horizontal: 'large', top: 'large' } : 'large'}
+            direction="row"
             // Height for template screen needs to be between medium and large
             // to maintain aspect ratio, so this is small + medium
-            height={template ? '576px' : undefined}
+            height={template ? '576px' : { min: '576px' }}
```

Comment:
Rather than hard code the `576px`, can we do the calculation for small + medium? That way if the theme changes base spacing, definitions of medium, small, etc. this would automatically adjust as well.

```
import {defaultProps} from 'grommet'
const { size } = defaultProps.theme.global;
const aspectHeight = `${parseInt(size.medium, 10) +
  parseInt(size.small, 10)}px`;
```

```
height={template ? aspectHeight : { min: aspectHeight }}
```

Reply:
Nice, yes let me adjust.

Reply:
Done!